pbm/pitr/pitr.go

```diff
@@ -62,6 +62,15 @@ func (s *Slicer) GetSpan() time.Duration {
 // the timeline (hence there are no restores after the most recent backup)
 func (s *Slicer) Catchup() error {
 	s.l.Debug("start_catchup")
+	cfg, err := s.pbm.GetConfig()
+	if err != nil {
+		return errors.Wrap(err, "get config")
+	}
+
+	if cfg.PITR.Unchecked {
+		return s.UncheckedCatchup()
+	}
```

Comment:
I'd rather move this level up. To leave the slicer decoupled with the config reading and other unrelated stuff.